rust/rbac-registration/src/registration/cardano/mod.rs

```diff
@@ -204,6 +205,21 @@ impl RegistrationChain {
             .get(role)
             .and_then(|rdr| rdr.encryption_key_from_rotation(rotation))
     }
+
+    /// Returns a list of role 0 stake addresses.
+    #[must_use]
+    pub fn stake_addresses(&self) -> Vec<StakeAddress> {
```

Comment:
What do you think to make it more general, and return stake_addresses by the provided role number ?
```suggestion
    pub fn stake_addresses(&self, role_number: &RoleNumber) -> Vec<StakeAddress> {
```

Reply:
I considered that too, but I'm not sure. Currently we are only using role 0 stake addresses and I don't know if we ever will be interested in addresses of other roles.

@bkioshn Any thoughts?

Reply:
Hmm, never discuss about this.
I think just role 0 would be enough and ignore other update on stake address for non role 0.

or do we have other use case for the stake address for non role 0? @stevenj

Reply:
We discussed this with @Mr-Leshiy and for now decided to rename the method to `role_0_stake_addresses`.